app/models/course_user.rb

```diff
@@ -1,5 +1,8 @@
 class CourseUser < ActiveRecord::Base
   belongs_to :user, inverse_of: :course_users
   belongs_to :course, inverse_of: :course_users
+  has_many :exp_records, class_name: 'Course::ExpRecord',
```

Comment:
<code> Course::ExpRecord.name </code>, Thus it will raise an exception when your class does not exist.